README.md

```diff
@@ -94,6 +94,23 @@ emitter.send(new CloudEvent({ specversion: Version.V03, source, type }))
   }).catch(console.error);
 ```
 
+#### CloudEvents
```

Comment:
nits:
- I'd probably put this as a H2 `##` given shouldn't be below the titles `Emitting Events` (H4) and `Receiving and Emitting Events` (H3). (The current header organization is a bit wonky tho)
- Change the title to `## CloudEvent Objects`? A bit more descriptive.